src/routes/auth/auth.service.ts

```diff
@@ -0,0 +1,89 @@
+import { Inject, Injectable, UnauthorizedException } from '@nestjs/common';
+import { VerifyAuthMessageDto } from '@/routes/auth/entities/schemas/verify-auth-message.dto';
+import { IAuthRepository } from '@/domain/auth/auth.repository.interface';
+import { IJwtService } from '@/datasources/jwt/jwt.service.interface';
```

Comment:
We should also move the access to this interface to be done via the `IAuthRepository` (as it's a Data Source).